### DISCUSSION TITLE
Identifying Deployed Version

### DISCUSSION THREAD

**Discussion by jmcrowe-illinois** [Azure/api-management-developer-portal#1515] [Releases]
Is there a way that I can identify the built-in Developer Portal version that is deployed on our APIM instance? 

I.e. which version we have deployed based on this...
https://github.com/Azure/api-management-developer-portal/releases

Thanks for any feedback.

**Comment by mikebudzynski** [comment]
There's currently no way to identify the deployed developer portal version. In most cases, all managed portals receive the release within 1-2 weeks of its announcement in the "Releases" section of the GitHub repository. For example, all managed portals are now running the 2.11.5 version. You are welcome to request this feature in the Issues section.
